client/transaction-pool/src/lib.rs

```diff
@@ -570,11 +571,17 @@ impl<PoolApi, Block> MaintainedTransactionPool for BasicPool<PoolApi, Block>
 					block_number,
 					Some(std::time::Duration::from_secs(60)),
 					Some(20.into()),
+					is_new_best,
```

Comment:
Why isn't the whole `maintain` thing run only if `is_new_best == true`? What kind of maintenance are we supposed to do when forks are imported?

Reply:
IDK :P I don't have written this code, but your comment made sense and I changed the code to only run on `is_new_best == true`